src/libstore/remote-store.cc

```diff
@@ -1084,6 +1084,19 @@ unsigned int RemoteStore::getProtocol()
     return conn->daemonVersion;
 }
 
+bool RemoteStore::isTrustedUser()
+{
+    auto conn(getConnection());
+    if (GET_PROTOCOL_MINOR(conn->daemonVersion) >= 35) {
+        conn->to << wopIsTrustedUser;
+        conn.processStderr();
+        return readInt(conn->from);
+    } else {
+      throw Error("daemon protocol %d.%d is too old (< 2.13) to ask whether we're a trusted user",
+          (GET_PROTOCOL_MAJOR(conn->daemonVersion) >> 8),
+          GET_PROTOCOL_MINOR(conn->daemonVersion));
```

Comment:
I actually don't think that printing the daemon protocol version is useful here since it's mostly an internal detail. It would be better to point to the minimal working Nix version (2.13 in that instance).

Reply:
I also felt that this was unnecessary, but just copied the wisdom/pattern from other parts of the code. I'll remove it.